src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationMethodCallLineWrap2.java

```diff
@@ -0,0 +1,59 @@
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation; //indent:0 exp:0
+
+/**                                                                           //indent:0 exp:0
+ * This test-input is intended to be checked using following configuration:   //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ * basicOffset = 4                                                            //indent:1 exp:1
+ * lineWrappingIndentation = 4                                                //indent:1 exp:1
+ * tabWidth = 4                                                               //indent:1 exp:1
+ * forceStrictCondition = true                                                //indent:1 exp:1
+ *                                                                            //indent:1 exp:1
+ */                                                                           //indent:1 exp:1
+
+public class InputIndentationMethodCallLineWrap2 {                            //indent:0 exp:0
+
+    InputIndentationMethodCallLineWrap2[] arr = {this};                       //indent:4 exp:4
+    InputIndentationMethodCallLineWrap2[][] arr2 = {{this}, {this}};          //indent:4 exp:4
+
+    InputIndentationMethodCallLineWrap2 obj() {                               //indent:4 exp:4
+        return this;                                                          //indent:8 exp:8
+    }                                                                         //indent:4 exp:4
+
+    InputIndentationMethodCallLineWrap2[] obj2() {                            //indent:4 exp:4
+        return arr;                                                           //indent:8 exp:8
+    }                                                                         //indent:4 exp:4
+
+    InputIndentationMethodCallLineWrap2[][] obj3() {                          //indent:4 exp:4
+        return arr2;                                                          //indent:8 exp:8
+    }                                                                         //indent:4 exp:4
+
+    void testMethod() {                                                       //indent:4 exp:4
+        InputIndentationMethodCallLineWrap2 obj =                             //indent:8 exp:8
+            new InputIndentationMethodCallLineWrap2();                        //indent:12 exp:12
+
+        obj                                                                   //indent:8 exp:8
```

Comment:
Please add cases with 2, 3, ... chained calls on the first line. Ex:

```
       obj.obj2()[0].obj()
            .obj()    

```

Please show violation and ok.